tests/system/View/ParserPluginTest.php

```diff
@@ -1,17 +1,22 @@
 <?php
 
-use CodeIgniter\View\Parser;
-
 class ParserPluginTest extends \CIUnitTestCase
 {
+	/**
+	 * @var \CodeIgniter\View\Parser
+	 */
 	protected $parser;
+	/**
+	 * @var \CodeIgniter\Validation\Validation
+	 */
 	protected $validator;
 
 	public function setUp()
 	{
 		parent::setUp();
 
 		\Config\Services::reset();
+		\Config\Services::autoloader()->initialize(new \Config\Autoload());
```

Comment:
What do you think about initialize the autoloader in the \Config\Services::reset() method? Cleans all, initialize the autoloader...